static/js/dashboard_routes.js

```diff
@@ -27,15 +27,15 @@ export function makeDashboardRoutes(browserHistory, store, onRouteUpdate) {
       <Provider store={store}>
         <Router history={browserHistory} onUpdate={onRouteUpdate}>
           <Route path="/" component={App}>
-            <Route path="dashboard" component={DashboardPage}/>
+            <Route path="dashboard" component={DashboardPage} />
             <Route path="profile" component={ProfilePage}>
-              <IndexRedirect to="personal"/>
-              <Route path="personal" component={PersonalTab}/>
+              <IndexRedirect to="personal" />
+              <Route path="personal" component={PersonalTab} />
               <Route path="education" component={EducationTab}/>
-              <Route path="professional" component={EmploymentTab}/>
-              <Route path="privacy" component={PrivacyTab}/>
+              <Route path="professional" component={EmploymentTab} />
+              <Route path="privacy" component={PrivacyTab} />
             </Route>
-            <Route path="/terms_of_service" component={TermsOfServicePage}/>
+            <Route path="/terms_of_service" component={TermsOfServicePage} />
```

Comment:
I agree that `<Foo prop="asdf" />` is more readable than `<Foo prop="asdf"/>`, is there an eslint rule we can turn on to enforce this?